packages/components/src/focal-point-picker/styles/focal-point-picker-style.ts

```diff
@@ -54,9 +56,32 @@ export const StyledUnitControl = styled( UnitControl )`
 	width: 100px;
 `;
 
+const deprecatedBottomMargin = ( {
+	__nextHasNoMarginBottom,
+}: FocalPointPickerControlsProps ) => {
+	return ! __nextHasNoMarginBottom
+		? css`
+				padding-bottom: 1em;
+		  `
+		: undefined;
+};
+
+const extraHelpTextMargin = ( {
+	hasHelpText = false,
+}: FocalPointPickerControlsProps ) => {
+	return hasHelpText
+		? css`
+				padding-bottom: 1em;
+		  `
+		: undefined;
+};
```

Comment:
I decided it was good to keep this extra bottom margin when there is a `help` text provided, because the whitespace is more balanced.